src/input/scroll.c

```diff
@@ -375,7 +368,7 @@ void scroll_get_delta(const mouse *m, pixel_offset *delta, scroll_type type)
     time_millis current_time = time_get_millis();
     time_millis time_delta = current_time - data.speed.last_time;
     data.speed.last_time = current_time;
-    if (time_delta > 17 && time_delta < MAX_SPEED_TIME_WEIGHT) {
+    if (time_delta > (int) FRAME_TIME && time_delta < MAX_SPEED_TIME_WEIGHT) {
```

Comment:
I think I missed it last time, but I don't think it's a good idea to rely on a 17ms frame time. For 120Hz monitors this will be 8.3ms, and it will be undefined for systems which don't support vsync.

Reply:
So I have changed the code (the branch has 3 commits with this fixed) but due to the outage, I guess it didn't sync the PR.

Anyway, I removed that `(int) FRAME_TIME`. It seems to be working properly, and now scroll speed is normalized even if the monitor's refresh rate is higher than 60hz.